proxy/api/src/service.rs

```diff
@@ -12,6 +12,8 @@ pub struct Environment {
     pub key: Option<coco::keys::SecretKey>,
     /// If set, we use a temporary directory for on-disk persistence.
     pub temp_dir: Option<tempfile::TempDir>,
+    pub coco_paths: coco::Paths,
+    pub key_store: Arc<dyn coco::keystore::KeyStore + Send + Sync>,
```

Comment:
Missing documentation

Reply:
Fixed in https://github.com/radicle-dev/radicle-upstream/pull/1153/commits/751ddd213d26ef187ab6d84ea8b83ef22562df21